### PR TITLE
chore(all): get all node references from a single source

### PR DESCRIPTION
**Pull request by biffgaut**
*Description of changes:*
Create a single location to define the node version and point all references to this location (in utils.ts). Set it. to 20 to start.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: 223b2c710ea4d50d7d90a25121f61079232d0bc9
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=883c474a-a55c-4704-9d7e-1f0edf190470%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by georgebearden — APPROVED** [review]
looks good to me

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: c0fbcf9ffdada9229ff8450aaf2484ad0089ad5c
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=60901850-02ca-43c9-9619-77664b4c9630%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/patterns/@aws-solutions-constructs/core/lib/utils.ts [hunk 1]
```diff
@@ -21,8 +21,12 @@ import { flagOverriddenDefaults } from './override-warning-service';
 import * as log from 'npmlog';
 import * as crypto from 'crypto';
 import * as cdk from 'aws-cdk-lib';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
 import { Construct } from "constructs";
 
+export const COMMERCIAL_REGION_LAMBDA_NODE_RUNTIME = lambda.Runtime.NODEJS_20_X;
+export const COMMERCIAL_REGION_LAMBDA_NODE_STRING = "nodejs20.x";
```
Comment (georgebearden) [inline source/patterns/@aws-solutions-constructs/core/lib/utils.ts:28]: This is a nice change across the entire repo!

**Review by georgebearden — APPROVED** [review]
looks good to me

**Pull request merged**